### DISCUSSION TITLE
Possible back-cover bug UM-EE - Overleaf

### DISCUSSION THREAD

**Discussion by Marcelo-Alves7** [joaomlourenco/novathesis#323] [Q&A]
Hello professor,

I updated my document to the latest version, and after compiling the UM-ee back cover it appears as follows:

![image](https://user-images.githubusercontent.com/98588105/160457312-0822dc97-e6d9-4890-89c0-35a3aa437c78.png)

I thought it might be a working status, but the bug persists even after changing to final.
Any way to correct and appear the normal information (Study Program, Advisors...)?

Thanks.

**Comment by joaomlourenco** [comment]
@Marcelo-Alves7  Does it happen with an "out of the box" copy/download of the template?

**Comment by joaomlourenco** [comment]
@Marcelo-Alves7  I cannot reproduce this behaviour!  Without further info I'll ignore this help request.
